code/modules/research/techweb/all_nodes.dm

```diff
@@ -586,7 +586,7 @@
 	display_name = "Botanical Engineering"
 	description = "Botanical tools"
 	prereq_ids = list("adv_engi", "biotech")
-	design_ids = list("diskplantgene", "portaseeder", "flora_gun", "hydro_tray", "biogenerator", "seed_extractor")
+	design_ids = list("diskplantgene", "portaseeder", "flora_gun", "hydro_tray", "biogenerator", "seed_extractor", "plantgenes") //VOIDCREW EDIT
```

Comment:
Don't label it as voidcrew edit, it will be confused with things that are being kept around after the rebase. Marking edits don't matter since we don't currently update to anything, and plant gene console is being deleted entirely by the rebase once it happens.

Reply:
we aren't keeping current botany? were going to the form of botany that takes way longer to actually progress? botany is already a dead job with very few uses tg botany would just kill it ever being used